source/release-notes/changelogs/backend/changelog-backend-2022.rst

```diff
@@ -0,0 +1,45 @@
+.. _backend_20220519:
+
+19 May 2022 Release
+~~~~~~~~~~~~~~~~~~~~~
+
+- Introduced option to encode HTTPS Endpoint responses as :ref:`EJSON or JSON <endpoint-return-types>`.
```

Comment:
FWIW, when I try to click this link, it's not doing anything. Not sure why.

Reply:
That ref doesn't exist in this branch, but it'll work when I merge in the sibling PR: https://github.com/mongodb/docs-app-services/pull/18
Mostly just didn't want to base this branch on that branch since that branch could still change in copy review.